src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -56,6 +58,18 @@ export class Conversations extends React.Component<Props, State> {
     })
   }
 
+  refreshConversations(callback) {
+    if (this.state.fetchingNextPage) {
+      return
+    }
+
+    this.setState({ fetchingNextPage: true })
+    this.props.relay.refetchConnection(10, () => {
```

Comment:
What does this effectively do? Does it reset the connection to only load the initial 10 (after previously have paginated in more pages)?

Reply:
What I can gather from the [docs](https://facebook.github.io/relay/docs/en/pagination-container.html) is that it refetches the 10 newest items in the connection, and that it's the [recommended way to force-fetch `PaginationContainer`s](https://facebook.github.io/relay/docs/en/api-cheatsheet.html#to-force-fetch-a-component) (as opposed to `loadMore`).

Reply:
Ok.